### PR TITLE
Test 1

### PR DESCRIPTION
**Pull request by Sedrak1992**

### CODE REVIEW

File: src/components/HomePage/LeftSidebar/index.jsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const LeftSidebar = () => {
           <img src="/facebook/img/269669879_3137994416483686_4685938482215500696_n.jpeg" />
         </div>
 
-        <h3 className="ms-3">Sedrak Qocharyan</h3>
+        <h3 className="ms-3">Sedrak Qocharyan 88888</h3>
```
Comment (gug007) [inline src/components/HomePage/LeftSidebar/index.jsx]: @Sedrak1992 88888 tex@ pti liner 77777

Comment (gug007) [reply src/components/HomePage/LeftSidebar/index.jsx:22]: test

Comment (gug007) [reply src/components/HomePage/LeftSidebar/index.jsx:22]: test 1

Comment (gug007) [reply src/components/HomePage/LeftSidebar/index.jsx:22]: test 2

File: src/components/HomePage/LeftSidebar/index.jsx [hunk 2]
```diff
@@ -19,7 +19,7 @@ const LeftSidebar = () => {
           <img src="/facebook/img/269669879_3137994416483686_4685938482215500696_n.jpeg" />
```
Comment (gug007) [inline src/components/HomePage/LeftSidebar/index.jsx:19]: test 1

Comment (gug007) [inline src/components/HomePage/LeftSidebar/index.jsx:19]: 12

**Review by gug007 — APPROVED** [review]

File: src/components/HomePage/LeftSidebar/index.jsx (hunk 1 above)
Comment (Sedrak1992) [reply src/components/HomePage/LeftSidebar/index.jsx]: inch tenc

Comment (gug007) [reply src/components/HomePage/LeftSidebar/index.jsx]: mer dizayn proyetov 77777 petka lini

Comment (gug007) [reply src/components/HomePage/LeftSidebar/index.jsx]: @Sedrak1992 tkanks 👍🏾

**Pull request merged**